lab_1/main.py

```diff
@@ -36,7 +47,15 @@ def calculate_frequencies(tokens: list) -> dict:
     e.g. tokens = ['weather', 'sunny', 'man', 'happy']
     --> {'weather': 1, 'sunny': 1, 'man': 1, 'happy': 1}
     """
-    pass
+    frequency = {}
+    if isinstance(tokens, list):
```

Comment:
Usually, there are checks for arguments and body function - separated and on the same level.
Try to get rid of nesting.